source/deployment/bulk-loading-data-format.rst

```diff
@@ -725,6 +751,30 @@ Fields of the User object
       <td align="center" valign="middle">No</td>
       <td align="center" valign="middle">No</td>
     </tr>
+    <tr class="row-odd">
+      <td valign="middle">use_markdown_preview</td>
+      <td valign="middle">bool</td>
+      <td>Enable previewing markdown formatting. Can have one the following values:<br>
```

Comment:
For the two new Markdown formatting objects, we refer to them slightly differently in the Account Settings > Advanced UI. Not sure if this matters.

![image](https://user-images.githubusercontent.com/13119842/46574026-4587a000-c96b-11e8-9640-95d0cc235e9f.png)